Update "Salesforce Rest API Connector Example" Doc (#505)
wso2/docs-mi: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md

If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.

## Set up the integration project
1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
2. Create a new `Salesforcerest` connection. 
    1. Go to `Local Entries` -> `Connections` and click on the `+` sign.
    2. Select  `Salesforcerest` connector.
        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-connection.png" title="Add new salesforce connection" width="800" alt="Add new salesforce connection"/>

    3. Use following values can to create the connection. 

>         - Connection Name - `SalesforceRestConnection`

Reviewer: Make the property names bold

If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.

> ## Setting up the Integration Project

Reviewer 2: ```suggestion
## Set up the integration project
```

1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 

> 

Reviewer 2: remove extra new line

2. Create a new Salesforcerest connection. 

>     1. Goto `Local Entries` -> `Connections` and click on the `+` sign. 

Reviewer 2: ```suggestion
    1. Go to `Local Entries` -> `Connections` and click on the `+` sign. 
```

    2. Select  `Salesforcerest` connector.
        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-connection.png" title="Add new salesforce connection" width="800" alt="Add new salesforce connection"/>

    3. Use following values can to create the connection. 

>         - Connection Name - SalesforceRestConnection

Reviewer 2: ```suggestion
        - Connection Name - `SalesforceRestConnection`
```

>         - Connection Type - init 

Reviewer 2: ```suggestion
        - Connection Type - `init` 
```

        - Client ID - Value of your client ID given when you registered your application with Salesforce.
        - API Version - v44.0
        - API URL - Value of the instance url.
        - Host Name - https://login.salesforce.com

### Add integration logic

Select Micro Integrator and click on `+` in APIs to create a REST API. Specify the API name as `salesforcerest` and API context as `/salesforcerest`.
   
   <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>

Create the sequence needed to create Salesforce object. We will create two defined sequences called `create.xml` and  `retrieve.xml` to create an account and retrieve data. 
Select `Sequences` under the project and click on the `+` sign to create a sequence. 
    <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-sequence.png" title="Adding a Sequnce" width="800" alt="Adding a Sequnce"/>

#### Configure a sequence for the create operation
Now follow the steps below to add configurations to the sequence.

1. Provide `create` as name and create the sequence. 
     

> 2. Set up the create operation.

Reviewer 2: ```suggestion
2. Set up the `create` operation.
```

        
        While invoking the API, the above two parameters values come as a user input.
    
    2. Select the created `create` sequence. 
    3. Add the `create` operation from the `Salesforcerest` connector. 

        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-create-operation.png" title="Add create operation" width="800" alt="Add create operations"/>
    

>     3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator/). Add the `Property` mediators into the Design pane as shown bellow.

Reviewer 2: ```suggestion
    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator/). Add the `Property` mediators into the design pane as shown bellow.
```

    You may need to update the value of the access token and make other such changes before deploying and running this project.

## Deployment
To deploy and run the project, please refer to the [build and run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.

> You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli). 

Reviewer: ```suggestion
You can further refer to the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli). 
```

            },
            "Id": "001dM00000HdszpQAB",
            "Name": "United Oil & Gas, UK"
        },
        {
            "attributes": {
                "type": "Account",
                "url": "/services/data/v44.0/sobjects/Account/001dM00000HdszqQAB"
            },
            "Id": "001dM00000HdszqQAB",
            "Name": "United Oil & Gas, Singapore"
        },
        {
            "attributes": {
                "type": "Account",
                "url": "/services/data/v44.0/sobjects/Account/001dM00000HdszrQAB"
            },
            "Id": "001dM00000HdszrQAB",
            "Name": "GenePoint"
        },
        {
            "attributes": {
                "type": "Account",
                "url": "/services/data/v44.0/sobjects/Account/001dM00000HdszsQAB"
            },
            "Id": "001dM00000HdszsQAB",
            "Name": "sForce"
        },
        {
            "attributes": {
                "type": "Account",
                "url": "/services/data/v44.0/sobjects/Account/001dM00000HemdeQAB"
            },
            "Id": "001dM00000HemdeQAB",
            "Name": "Sample Account for Entitlements"
        }
    ]
}
```

> ## What's Next

Reviewer: ```suggestion
## What's next
```
